samples/react-native/src/App.tsx

```diff
@@ -37,6 +37,10 @@ import HeavyNavigationScreen from './Screens/HeavyNavigationScreen';
 import WebviewScreen from './Screens/WebviewScreen';
 import { isTurboModuleEnabled } from '@sentry/react-native/dist/js/utils/environment';
 
+if (typeof setImmediate === 'undefined') {
+  require('setimmediate');
+}
```

Comment:
Why is this required?

Reply:
This is a workaround for the [set `setImmediate` error](https://github.com/getsentry/sentry-react-native/pull/4207#issuecomment-2462139171) on iOS mentioned before. I haven't figured out the root of the issue.

Reply:
Aaa, my bad, yes, you described in the comments above.